markov.py

```diff
@@ -61,6 +61,16 @@ def remove_messages(message):
 
 
 @bot.message_handler(func=lambda m: True)
+def on_message(message):
+    update_model(message)
+    try:
+        if message.entities[0].type == 'mention':
```

Comment:
I think it would be better if this was a list; example:
```python
entities = [u.type for u in message.entities]
if 'mention' in entities:
....
```

and then you can remove the try except block

Reply:
by the way, I think just the 69 line is enough, `if f'@{bot.get_me().username}' in message.text:` why is it necessary to list the entities?

Reply:
Sorry about that, the change was already made but i forgot to push it.

The type will be a mention even when a different user/bot is mentioned